.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,18 @@
+on: [pull_request]
+
+jobs:
+  build-chain:
+    runs-on: ubuntu-latest
+    name: Build Pull Request
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: Ginxo/github-action-build-chain@master
```

Comment:
why Ginxo here?

Reply:
@mbiarnes because is the github action tool I created for handling the build chain
https://github.com/Ginxo/github-action-build-chain